src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = (
+  { person: { name, age, sex, isMarried, partnerName } },
+) => {
```

Comment:
```suggestion
export const Person = ({ person }) => {
  const { name, age, sex, isMarried, partnerName } = person ;
```
Avoid nesting destructing, it's difficult to read. You can destruct `person` from props in parameters. And then other values from this person